HotlineIO/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>1.3.0</string>
```

Comment:
Good question! To avoid confusion a deal with such things, our wrapper needs to have its own version. I guess we need to release 1.0 instead of 1.3.0